encord/storage.py

```diff
@@ -947,6 +973,21 @@ def _list_items(
         for item in paged_items:
             yield StorageItem(api_client, item)
 
+    @staticmethod
+    def _patch_multiple_folders(
+        api_client: ApiClient,
+        folder_patches: Dict[str, PatchFolderPayload],
+    ) -> List[orm_storage.StorageFolder]:
+        return api_client.patch(
+            "storage/folders/patch-bulk",
+            params=None,
+            payload=orm_storage.PatchFoldersBulkPayload(folder_patches=folder_patches),
+            result_type=Page[orm_storage.StorageFolder],
+        ).results
+
+    def _set_orm_item(self, orm_item: orm_storage.StorageItem) -> None:
```

Comment:
I think that this is unused. Also this is StorageFolder as opposed to StorageItem and otherwise doesn't have a 
```
self._orm_item
```
attribute.

Reply:
https://github.com/encord-team/cord-backend/pull/3355